src/explore-education-statistics-admin/src/pages/release/datablocks/components/FeaturedTablesTable.tsx

```diff
@@ -0,0 +1,197 @@
+import Link from '@admin/components/Link';
+import DroppableArea from '@admin/components/DroppableArea';
+import DraggableItem, { DragHandle } from '@admin/components/DraggableItem';
+import { FeaturedTable } from '@admin/services/featuredTableService';
+import { ReleaseDataBlockSummary } from '@admin/services/dataBlockService';
+import {
+  releaseDataBlockEditRoute,
+  ReleaseDataBlockRouteParams,
+} from '@admin/routes/releaseRoutes';
+import Button from '@common/components/Button';
+import ButtonText from '@common/components/ButtonText';
+import FormattedDate from '@common/components/FormattedDate';
+import reorder from '@common/utils/reorder';
+import useToggle from '@common/hooks/useToggle';
+import React from 'react';
+import { generatePath } from 'react-router';
+import orderBy from 'lodash/orderBy';
+import { DragDropContext, Droppable } from 'react-beautiful-dnd';
+
+interface Props {
+  canUpdateRelease: boolean;
+  dataBlocks: ReleaseDataBlockSummary[];
+  featuredTables: FeaturedTable[];
+  publicationId: string;
+  releaseId: string;
+  onDelete: (dataBlock: ReleaseDataBlockSummary) => void;
+  onReorder: (featuredTables: FeaturedTable[]) => void;
```

Comment:
Seems like there's a bit of jank when this handler calls. Think it takes a little bit of time for React Query to actually update, so it's fairly noticeable when reordering.

Think we can smooth this out by using local state instead i.e.

```tsx
const [currentFeaturedTables, setCurrentFeaturedTables] = useState(
  featuredTables,
);

useEffect(() => {
  setCurrentFeaturedTables(featuredTables);
}, [featuredTables]);
```